### PR TITLE
[Merged by Bors] - chore: bump aesop

### PR DESCRIPTION
**Pull request by kim-em**
This requires removing some rules that `aesop` rejects. They aren't needed, and in fact I think were never doing anything?

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Continuity.lean [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Mathlib.Algebra.Group.Defs
 We define the `continuity` tactic using `aesop`. -/
 
 attribute [aesop (rule_sets [Continuous]) unfold norm] Function.comp
-attribute [aesop (rule_sets [Continuous]) unfold norm] npowRec
```
Comment (eric-wieser) [inline Mathlib/Tactic/Continuity.lean:16]: Maybe worth a TODO comment (here and in the other file): `npowRec` is an implementation detail, and certainly not something aesop should know about.

**Comment by eric-wieser** [comment]
bors d+

Thanks! I don't really understand the aesop changes, but the `npowRec` stuff is already wrong, and the Sym2 stuff is pretty unimportant (and arguably those lemmas should be private anyway)

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
bors p=10

**Pull request closed without merging**